### ISSUE TITLE
Simplify the UI, drop the categories

### ISSUE THREAD

**Issue by dardoor** [PaulWoitaschek/Voice#984]
I think Voice went the wrong way, against its minimalist principles, when the Current/Not Started Yet/Completed were added.
Those categories and the "Show All" buttons mostly just take up space and contribute to visual clutter.
It's made worse by the fact that it can't always correctly categorize audiobooks, or even if it does, the categorization might not be functionally meaningful for the user (Completed vs Not Started Yet).

I suggest:
1. Drop all the categories (I mean the visible labels).
2. Always show all the audiobooks, in alphabetical order.
3. Show the currently playing ones at the top. No need to label them "currently playing", the progress bar indicator should already make this evident (maybe you can also add some color coding/highlight).
4. Let the user long press on an audiobook and manually mark it as "completed", so that it drops back down from the top.

**Comment by seathwolf** [comment]
I have to disagree with you, I like that the I have my completed and current books in separate area. I really do not want the clutter of all book in same space. I have a large library and that would be mess.
But I do agree that there could be better way to do that. For example I used app called Retro Music before I found Voice, and what I really liked about it was the bottom tab navigation. I believe this would work for Voice too. Simple and clean.

**Comment by jtagcat** [comment]
Instead of alphabetical, also offer (by default) most recently played and most recently added (new books) (combined?).

Not only manually mark it as 'completed', mark it as 'uncompleted'. Mark as 'abandoned' could also be a thing (stopped reading).

**Comment by jtagcat** [comment]
Related: #955
